sdk/sdk/pom.xml

```diff
@@ -44,7 +44,6 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-aws</artifactId>
-      <version>${hadoop.version}</version>
```

Comment:
please remove "build-all" profile also as we build all modules now

Reply:
done